recipes/sail/all/conanfile.py

```diff
@@ -1,9 +1,10 @@
-from conan.tools.files import rename
-from conans import ConanFile, CMake, tools
-import functools
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, copy, get, rename, rmdir
+from conan.tools.microsoft import is_msvc
 import os
 
-required_conan_version = ">=1.43.0"
+required_conan_version = ">=1.51.0"
```

Comment:
```suggestion
required_conan_version = ">=1.53.0"
```
Required by self.options.rm_safe